src/main/java/seedu/address/model/appointment/Medicine.java

```diff
@@ -27,7 +27,7 @@ public class Medicine {
     public Medicine(String medicine) {
         requireNonNull(medicine);
         checkArgument(isValidMedicine(medicine), MESSAGE_CONSTRAINTS);
-        value = medicine;
+        value = medicine.toLowerCase();
```

Comment:
Is there a particular reason why medicine in lower case? Because for other attributes like email or address (which are also string attributes), there don't do this.